application/lib/view/widgets/custom_bottom_sheet_scaffold.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:application/utils/colors.dart';
+import 'package:application/utils/styles/styles.dart';
+import 'package:application/utils/translation_en.dart';
+import 'package:application/view/screens/add_product_screen.dart';
+import 'package:flutter/material.dart';
+
+class CustomBottomSheetScaffold extends StatelessWidget {
```

Comment:
A Widget neve célszerű ha elmondja azt is, hogy mi a célja. Pl LoadingIndicator, HomeScreen, BottomNavigation, stb. Ebből a névből egyenlőre csak az látszik, hogy valaminek az alján egy sheet. Jobb lehet pl a CustomNavigationSheet.